### DISCUSSION TITLE
oXygen versions in the TEI community

### DISCUSSION THREAD

**Discussion by ebeshero** [TEIC/TEI#2122] [Q&A]
What version of &lt;oXygen/&gt; are you using?

**Comment by peterstadler** [comment]
23.1

**Reply by ebeshero** [reply]
Same here!

**Comment by lujessica** [comment]
i was using 23.0, but this just reminded me to update! so, 23.1 now.

**Comment by jamescummings** [comment]
Depends on the computer to be honest. I am using 21.1 on this laptop.

**Comment by martindholmes** [comment]
23.1.

**Comment by sydb** [comment]
As with @jamescummings it depends on the machine I am on. My main desktop here at home (GNU/Linux) is currently running 23.0, but like @lujessica I am reminded to upgrade soon.
My desktop at work (also GNU/Linux) is probably running a significantly older version, but since I haven’t used oXygen on that machine for 1 year (to the day), and probably won’t for many months, it does not matter much. :frowning_face:

**Reply by PFSchaffner** [reply]
On this Lenovo Win10 laptop? none, it takes too much space on my constrained SSD. On my ancient Win7 notebook with the dead keyboard? I think maybe 18. On the office machine I haven't been able to get to for a year? Maybe 19??. Most recent license and installer in hand? 21.1.

**Comment by lb42** [comment]
Always the most recent one I can get. On Linux desktop and Linux laptop and even... my raspberry pi


Reluctantly using Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by lb42** [comment]
Ah wait, I just checked and the laptop is still running an older 
version. So put me down for the following

- 22.1 on the linux desktop (but I just downloaded the upgrade)

- 20.0 on the linux  laptop

- 23.0 on the raspberry pi !

totally oxygenated

Lou

**Comment by tillgrallert** [comment]
21.1 on macOS (laptop) because later versions have an issue inherited from the bundled Java version that causes a crash when the keyboard language is changed while oXygen is not in focus.
